api/XRSession.json

```diff
@@ -1269,6 +1275,156 @@
           }
         }
       },
+      "squeeze_event": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/XRSession/squeeze_event",
+          "spec_url": "https://immersive-web.github.io/webxr/#eventdef-xrsession-squeeze",
+          "description": "<code>squeeze</code> event",
+          "support": {
+            "chrome": {
+              "version_added": "83"
```

Comment:
Confirmed by https://storage.googleapis.com/chromium-find-releases-static/1f8.html#1f86cd04485dfa24dfddd138a3b2a8d4c93639c0 (which says 82, but that release was skipped)